src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public abstract class Figure implements Figuring,Drawable {
+    private Color color;
+    private double area;
+
+    public Color getColor() {
+        return color;
+    }
+
+    public void setColor() {
+        int randomGenaratedIndex = new Random().nextInt(Color.values().length);
+        this.color = Color.values()[randomGenaratedIndex];
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: ";
+    }
```

Comment:
let's remove this method

Reply:
in Readme file was example of return " Figure: triangle, area: 12.5 sq. units, hypotenuse: 7.1 units, color: yellow". So "Figure:" should be in all of Figures that's why i put  "Figure:" is global method and add each unique characterictics in by each concrete file. If i delete this i have in each figure file  draw() add "Figures:"... I believed that part of code that is general for each class we have put in level up class , in my example in abstarct class Figure. Am i wrong?

Reply:
done